parachains/runtimes/assets/statemine/src/xcm_config.rs

```diff
@@ -96,7 +96,7 @@ pub type FungiblesTransactor = FungiblesAdapter<
 	AccountId,
 	// We only want to allow teleports of known assets. We use non-zero issuance as an indication
 	// that this asset is known.
-	parachains_common::impls::NonZeroIssuance<AccountId, Assets>,
+	LocalMint<parachains_common::impls::NonZeroIssuance<AccountId, Assets>>,
```

Comment:
I actually don't understand why this is `LocalMint`? This is Assets 1 as we discussed, yeah? Then it should be receiving native parachain tokens, and hence should be `NonLocalMint`.

Reply:
DOT/KSM is not ever going to be represented in the Assets pallet; they're instead going to be represented in the Balances pallet, and hence won't even use `FungiblesAdapter` but instead `CurrencyAdapter`.

Reply:
Oh wait nevermind, this is correct: instance 1 assets are integer-based, and so can be created locally by any user.